test/integration/branch_int_test.go

```diff
@@ -29,7 +29,7 @@ func (suite *BranchIntegrationTestSuite) TestBranch_List() {
 	cp.Expect("To switch to another branch,")
 	cp.ExpectExitCode(0)
 
-	ts.PrepareProject("ActiveState-CLI/small-python", "")
+	ts.PrepareProject("ActiveState-CLI/small-python", "00000000-0000-0000-0000-000000000000")
```

Comment:
Is it worth allowing this to be empty and default to this UUID in `PrepareProject()`? Technically, this UUID does not exist for this project, so leaving it empty from a visual perspective makes sense to me.